test/stack_status_test.go

```diff
@@ -3,7 +3,6 @@ package tests
 import (
```

Comment:
Code was moved into `suite_test.go` since the `makeFixtureIntoRepo` function is used by most tests.